WordPress/Jetpack/Resources/release_notes.txt

```diff
@@ -1,5 +1,11 @@
-* [*] Site previews: Reduced visual flickering when previewing sites and templates. [#17861]
-* [*] Stats: Scroll to new Insights card when added. [#17894]
-* [*] Add "Copy Link" functionality to Posts List and Pages List [#17911]
-* [***] Enables the ability to use and create WordPress.com sites, and enables the Reader tab. [#17914, #17948]
-* [*] Block editor: Additional error messages for media upload failures. [#17971]
+Three, two, one, liftoff—the Jetpack app now allows you to use and create WordPress.com sites and use the same Reader features as the WordPress app. The sky’s the limit.
+
+Almost as exciting, notifications and comments now support .gif files. (Unfortunately, this update doesn’t fix the issue of whether it’s pronounced “gif” or “jif.”)
+
+We fixed the flickering effect in Preview mode when you’re toggling between different screen sizes or orientations.
+
+When you add a new Insights card on the Stats screen, your screen view will jump down to show your brand-new card.
+
+You can now copy links to published, scheduled, and draft posts and pages using the Copy Link button above the Trash button. We also translated the button text into Turkish. Delightful.
```

Comment:
> We also translated the button text into Turkish. Delightful.

Now I'm hungry for Turkish delights 😋 

Btw, I'm not sure why we specifically mention Turkish translation here while it will be translated in many languages, not just `tr`.
I think it's because the contributor initially provided the Turkish translation by directly adding the string to the `tr.lproj/Localizable.strings` in their PR? But that ends up being overridden by our GlotPress translation pipeline anyway, and as you can see by looking at the various `*.lproj/Localizable.strings` in that PR, yes `tr.lproj` ended up being part of the translations already approved in GlotPress for "Copy Link", but they are far from the only language for which we got a translation for that new string.

I'd thus suggest to remove that last sentence from this paragraph of the notes.